### PR TITLE
Add Kebechet administrator

### PR DESCRIPTION
**Pull request by saisankargochhayat**
## Related Issues and Dependencies
Related - https://github.com/thoth-station/workflow-helpers/issues/148
Added Kebechet administrator workflow templates
Would need access to database, which I have added in the argo workflow.

### CODE REVIEW

**Review by harshad16 — COMMENTED** [review]
lgtm :+1: 

please include them here: https://github.com/thoth-station/thoth-application/blob/d7cacf87463cbf837c89bcb49bc79a9b6ca8b1a2/kebechet/base/kustomization.yaml#L9

and add, the following snippet to stage under: https://github.com/thoth-station/thoth-application/blob/d7cacf87463cbf837c89bcb49bc79a9b6ca8b1a2/kebechet/overlays/stage/kustomization.yaml#L35
```
  - path: put-into-infra-namespace.yaml
    target:
      group: template.openshift.io
      version: v1
      kind: Template
      name: kebechet-administrator
```

**Comment by saisankargochhayat** [comment]
@harshad16 thanks, I have added them to kustomize now.

File: kebechet/base/kustomization.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ resources:
   - openshift-templates/kebechet.yaml
   - argo-workflows/kebechet-run-url.yaml
   - openshift-templates/kebechet-run-url.yaml
+  - openshift-templates/kebechet-administrator.yaml
```
Comment (harshad16) [inline kebechet/base/kustomization.yaml:10]:
```suggestion
  - argo-workflows/kebechet-administrator.yaml
  - openshift-templates/kebechet-administrator.yaml
```

Comment (saisankargochhayat) [reply kebechet/base/kustomization.yaml:11]: Thanks

**Review by harshad16 — APPROVED** [review]
lgtm :+1:

**Pull request merged**